Libraries/LibWeb/CMakeLists.txt

```diff
@@ -366,8 +366,8 @@ set(SOURCES
     FileAPI/FileList.cpp
     FileAPI/FileReader.cpp
     FileAPI/FileReaderSync.cpp
-    Gamepad/Gamepad.cpp
-    Gamepad/NavigatorGamepad.cpp
```

Comment:
:thinking: a bit confused by this because we have other namespace/class repeats like this, e.g. `Clipboard::Clipboard` and `DOMURL::DOMURL`. Is something in the bindings not using the interface's fully qualified name?

Reply:
Turns out generate_to_cpp did not account for type names conflicting with namespaces. `Clipboard::Clipboard` was working because it was only used in impl_from, which takes account of this.